src/main/java/bookface/logic/parser/add/AddBookArgumentsParser.java

```diff
@@ -36,7 +36,7 @@ public AddBookCommand parse(String args) throws ParseException {
         Author author = ParserUtil.parseAuthor(argMultimap.getValue(PREFIX_AUTHOR).get());
         Title title = ParserUtil.parseTitle(argMultimap.getValue(PREFIX_TITLE).get());
 
-        Book book = new Book(title, author);
+        Book book = new Book(title, author, "Available");
```

Comment:
I don't think we should have `loanStatus` in the `Book` constructor. Whenever we create a new instance of a `Book`, it should always by default be available, so the logic to add a loanee should always come separately and later.

So to represent this idea of being possibly on loan, we can use an `Optional` for the loanee. So this avoids the possibility of having a NullPointerException and the need for additional `isLoaned` and `loanStatus` fields.

Reply:
I agree. My first iteration in fact does not have loanStatus in the Book constructor but I could not figure out how to implement it properly and it was left in.